stacks-doctor/src/cli.rs

```diff
@@ -0,0 +1,74 @@
+use core::fmt::{self, Formatter};
+use std::{
+    fmt::{Debug, Display},
+    path::PathBuf,
+    str::FromStr,
+};
+
+use clap::{Parser, Subcommand};
+
+#[derive(Clone, Debug)]
+pub enum Network {
+    Mainnet,
+    Testnet,
+}
+
+impl Display for Network {
+    fn fmt(&self, f: &mut Formatter) -> fmt::Result {
+        write!(f, "{:?}", self)
+    }
+}
+
+impl FromStr for Network {
+    type Err = String;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s.to_ascii_lowercase().as_str() {
+            "mainnet" => Ok(Network::Mainnet),
+            "testnet" => Ok(Network::Testnet),
+            _ => Err(format!("Could not parse Network: {}", s)),
+        }
+    }
+}
+
+#[derive(Parser, Clone, Debug)]
+pub struct BurnsArgs {
+    // How many recent blocks to take into account
+    #[arg(short, long, default_value_t = 1000)]
+    pub blocks: u64,
+}
+
+#[derive(Subcommand, Clone, Debug)]
+pub enum Commands {
+    /// Analyze miner
+    #[command(subcommand)]
+    Analyze,
```

Comment:
Nit: This is not another subcommand right?
```suggestion
    Analyze,
```

Reply:
No, it was a subcommand before but I've simplified it, for now.